analyses/fusion_filtering/README.md

```diff
@@ -0,0 +1,47 @@
+## Fusion prioritization
+
+**Module author:** Krutika Gaonkar ([@kgaonkar6](https://github.com/kgaonkar6))
+Jaclyn Taroni ([@jaclyn-taroni] (https://github.com/jaclyn-taroni))
+Jo Lynne([@jharenza](https://github.com/jharenza))
+Komal Rathi ([@komalsrathi] (https://github.com/komalsrathi))
+
+
+This analysis will perform artifact filtering and annotation on fusion calls from STARfusion and Arriba fusion callers to prioritize putative oncogenic fusions. 
+Briefly, we considered all inframe and frameshift fusion calls with a minimum of 1 junction reads and at least one gene partner expressed (TPM > 1) to be true calls. 
```

Comment:
```suggestion
We considered all inframe and frameshift fusion calls with a minimum of 1 junction reads and at least one gene partner expressed (TPM > 1) to be potential true calls. 
```

Reply:
I also think this should be `FPKM > 1` rather than `TPM > 1`?